seqr/views/utils/export_utils.py

```diff
@@ -25,6 +28,9 @@ def export_table(filename_prefix, header, rows, file_format='tsv', titlecase_hea
     Returns:
         Django HttpResponse object with the table data as an attachment.
     """
+    def _to_str(s):
+        return str(s, 'utf-8', errors = 'ignore') if not isinstance(s, str) else s
```

Comment:
For Python 3, `str` objects can't be decoded. Therefore, it is necessary to check `isinstance(s, str)`.